launcher/main.cpp

```diff
@@ -51,6 +51,15 @@ int MAIN_EXPORT main(int argc, char * argv[])
 	MainWindow mainWindow;
 	mainWindow.show();
 
+#ifdef VCMI_ANDROID
+	// changing language causes window to increase height over the bounds, force the height back to a proper value
+	// TODO: check in 5.15.14+ / 6 if the hack is still needed
+	QObject::connect(vcmilauncher.focusWindow(), &QWindow::heightChanged, []{
+		auto appWindow = qApp->focusWindow();
+		appWindow->resize(appWindow->width(), appWindow->screen()->availableSize().height());
```

Comment:
Can you do the same thing on width?
Height is now okay. But with still have the problem. :D

Reply:
sure. I faced only height issue on modern android (just like everybody else), didn't think that width would also be a problem :)

Reply:
done

Reply:
fixed. :)

Reply:
FYI seems it's no longer an issue in 5.15.14, so I dropped this hack

Reply:
@kambala-decapitator We should introduce it again. The bug is not gone. :(
Change between some languages (e.g. french) after some constellations (e.g. german -> french) the button is partially out of screen. Even for the next pages.

Reply:
Ok, let's add back. Couldn't reproduce when switching even to French.

Reply:
@kambala-decapitator Maybe it has to do with the control bar of android...

https://github.com/vcmi/vcmi/assets/13953785/6d276734-89a9-4555-b5ac-fa49cd35a4db

Reply:
That's possible. Qt 5 made various changes to computing available screen area, maybe some are available only in 5.15.15+.

That's why I want to switch android 64-bit to Qt 6 to gain all the latest fixes to modern Android versions.

Reply:
#4114